qtconsole/tests/test_jupyter_widget.py

```diff
@@ -70,7 +72,21 @@ def test_other_output(self):
         ))
 
         # Check proper syntax highlighting
-        self.assertEqual(document.toHtml(), (
+        if IS_QT6:
```

Comment:
You missed this comment from my previous review:

Please use

```python
if IS_QT6:
    html = ...
else:
    html = ...
```

instead of trying to pack everything in an inline if.

Reply:
If I put something like: 

```python
if IS_QT6:
    html = '<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">\n'
    '<html><head><meta name="qrichtext" content="1" /><meta charset="utf-8" /><style type="text/css">\n'
    'p, li { white-space: pre-wrap; }\n'
else:
    html = '<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">\n'
    '<html><head><meta name="qrichtext" content="1" /><style type="text/css">\n'
    'p, li { white-space: pre-wrap; }\n'
```

So html only takes the value of the first line and not all three lines. 

If you wanted to keep that format you would have to add `\` to the end of each line: 



```python
if IS_QT6:
    html = '<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">\n' \
    '<html><head><meta name="qrichtext" content="1" /><meta charset="utf-8" /><style type="text/css">\n' \
    'p, li { white-space: pre-wrap; }\n'
else:
    html = '<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">\n' \
    '<html><head><meta name="qrichtext" content="1" /><style type="text/css">\n' \
    'p, li { white-space: pre-wrap; }\n'
```

Is that what you want? I prefer to keep using `()`. So in the end it is your decision, which option do you want?

Reply:
> If you wanted to keep that format you would have to add \ to the end of each line:

Nop, you can also enclose the strings in parenthesis:

```python
if IS_QT6:
    html = (
        '<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">\n'
        '<html><head><meta name="qrichtext" content="1" /><meta charset="utf-8" /><style type="text/css">\n'
        'p, li { white-space: pre-wrap; }\n'
    )
else:
    html = (
        '<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">\n'
        '<html><head><meta name="qrichtext" content="1" /><style type="text/css">\n'
        'p, li { white-space: pre-wrap; }\n'
    )
```

That avoids the continuation lines and gives the result I'd like to see.